### DISCUSSION TITLE
Download everything after login-in?

### DISCUSSION THREAD

**Discussion by Ilovemyhous** [abraunegg/onedrive#1931] [Q&A] [answered]
Hello!
I have a quick question before installing this tool.
If I sign in to my (Office 365/Business) OneDrive account, which has 100+GB of data; will it download everything at once, or are they only available when I'm online?
Also, sorry if it was already asked, and answered before. ^^'

**Comment by abraunegg** [comment, marked as answer]
@Ilovemyhous 
The client will only download data when you are online, and the data will be available when you are offline as this is making a local copy of your online data.

**Reply by Ilovemyhous** [reply]
Okay, thanks for the answer!

**Comment by abraunegg** [comment]
Please stop closing

**Reply by Ilovemyhous** [reply]
Sorry, thought it was a bug on my side.
